test/e2e/main.go

```diff
@@ -20,6 +22,11 @@ type Test struct {
 func main() {
 	logger := log.New(os.Stdout, "test-e2e", log.LstdFlags)
 
+	latestVersion, err := testnet.GetLatestVersion()
+	testnet.NoError("failed to get latest version", err)
+
+	logger.Println("Running major upgrade to v2 test", "version", latestVersion)
```

Comment:
I think the tests themselves should get the latest version if they need them. We don't need to pass them through

Reply:
yes that make sense, will revert this.